osdk/src/commands/build/bin.rs

```diff
@@ -157,9 +157,7 @@ fn install_setup_with_arch(
     cmd.arg("install").arg("linux-bzimage-setup");
     cmd.arg("--force");
     cmd.arg("--root").arg(install_dir.as_ref());
-    cmd.arg("--git").arg(crate::util::ASTER_GIT_LINK);
-    // FIXME: Uses a fixed tag instaed of relies on remote branch
-    cmd.arg("--tag").arg("v0.5.1");
+    cmd.arg("--path").arg("../../../ostd/libs/linux-bzimage/setup");
```

Comment:
@StevenJiang1110 Is it ok to use the local one when taking into consideration that OSDK is published on crates.io?

Reply:
It is a temporary testing solution. Should be fixed.

Reply:
Yes, it's only used for testing, but cannot be used for published OSDK.

So I suggest OSDK should change the place where it tries to download `linuz-bzimage-setup`. OSDK should depends on local `linux-bzimage-setup` when running test and depends on remote  `linux-bzimage-setup` when publishing.  

For example, it can depend on the environmental variable.

```rust
if let Ok(_) = std::env::var("OSDK_INTEGRATION_TEST") {
    cmd.arg("--path").arg("../../../ostd/libs/linux-bzimage/setup");
} else {
    cmd.arg("--git").arg(crate::util::ASTER_GIT_LINK);
    cmd.arg("--tag").arg("v0.5.1");
}
```

The env variable should be set when running each OSDK test.

Reply:
@StevenJiang1110 So we may start publishing `linux-bzimage/setup`? Can you include this in #1009 ?

Reply:
> @StevenJiang1110 So we may start publishing linux-bzimage/setup? Can you include this in https://github.com/asterinas/asterinas/issues/1009 ?

Yes, it'a alreay published in PR #1018. [crates.io link](https://crates.io/search?q=linux-bzimage-setup). So the OSDK in PR #1018 won't depend on git tag, but on crates.io.

Reply:
Is `linux-bzimage` suitable for use by other kernels? I have seen code that hardcodes the text `ASTER`. If it is not suitable for general usage, prefix the crate name with `ostd-`. We want to use `crates.io` in a responsible way, without polluting its namespace and bothering its users with crates that are useless to them.

Reply:
> Is `linux-bzimage` suitable for use by other kernels?

It should be suitable. But currently Asterinas don't support relocation (yet) and there would be hard coded assumptions in the bzimage crate. We can work further for using it for other kernels.